datasetinsights/io/downloader/gcs_downloader.py

```diff
@@ -0,0 +1,30 @@
+from datasetinsights.io.downloader.base import DatasetDownloader
+from datasetinsights.io.gcs import download_folder_from_gcs
+
+
+class GCSDownloader(DatasetDownloader, protocol="gs://"):
+    """ This class is used to download data from GCS
+    """
+
+    def __init__(self, **kwargs):
+        """
+
+        Args:
+            access_token: Access token to be used to authenticate to
+            unity simulation for downloading the dataset
+        """
```

Comment:
`__init__` method should initialize a GCS client that have the right credential (read from default credential location or environment variable). In the future, we might want to support initialization using a path to GCS service account as well as optional parameter such as GCP project.

https://googleapis.dev/python/storage/latest/client.html#module-google.cloud.storage.client